test/tracing/testdata/emojivoto.yaml

```diff
@@ -0,0 +1,201 @@
+---
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: emoji
+---
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: voting
+---
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: web
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  creationTimestamp: null
+  name: emoji
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: emoji-svc
+  strategy: {}
```

Comment:
Just a nit. For good measure, you can jsut remove these empty fields as we already removed then in the other yaml files as well. Not that it would cause failures.